mint.json

```diff
@@ -40,6 +40,14 @@
       "name": "Components",
       "url": "content"
     },
+    {
+      "name": "Advanced",
+      "url": "advanced"
+    },
```

Comment:
Can we move advanced to the end? A fair number of folks use the integrations

Reply:
done